### PR TITLE
Miscellaneous minor code style fixes

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

File: reporter/src/main/kotlin/reporters/ReportTableModel.kt [hunk 1]
```diff
@@ -211,13 +211,20 @@ data class ReportTableModel(
...
-        val warningCount = unresolvedIssues[Severity.WARNING].orEmpty().size
-        val hintCount = unresolvedIssues[Severity.HINT].orEmpty().size
+        val errorCount: Int
+        val warningCount: Int
+        val hintCount: Int
+
+        init {
+            val unresolvedIssues = rows.flatMap {
+                it.analyzerIssues.flatMap { (_, issues) -> issues } +
+                        it.scanIssues.flatMap { (_, issues) -> issues }
```
Comment (sschuberth) [inline reporter/src/main/kotlin/reporters/ReportTableModel.kt:221]: Nit: This still would have fitted into the previous line.

Comment (fviernau) [reply reporter/src/main/kotlin/reporters/ReportTableModel.kt:221]: Are you aware that this is already merged? If so, what's your intention behind commenting nits on already merged PRs? I'm not quite sure how to interpret these comments.

Comment (sschuberth) [reply reporter/src/main/kotlin/reporters/ReportTableModel.kt:221]: > Are you aware that this is already merged?

Yes, that's why I wrote "would have fitted" in past tense.

> If so, what's your intention behind commenting nits on already merged PRs?

The ask to watch more closely for these kinds of (minor) formatting issues in future PRs.

Comment (fviernau) [reply reporter/src/main/kotlin/reporters/ReportTableModel.kt:221]: Thanks for explaining.

**Pull request merged**